examples/upload-images-application-command/go.mod

```diff
@@ -0,0 +1,5 @@
+module upload-images-application-command
+
+go 1.16
+
+require github.com/andersfylling/disgord v0.32.3 // indirect
```

Comment:
can you setup a replace directive, like the other examples. Then the CI will detect if a PR breaks any functionality